### PR TITLE
:memo: Document better content reuse methods

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

To have simplier ways to reuse content.

## What's changed

Added more instructions for reusing content.

### CODE REVIEW

File: contributing/markup-format.md [hunk 1]
```diff
@@ -218,9 +218,58 @@ Property    | Description
...
+
+Use the variable in the file:
+
+```markdown
+The file's name is: {{ .Get "FileName" }}
+```
+
+### Inner content
+
+For longer content specific to each file, you can put content inside the shortcode:
```
Comment (trolologuy) [inline contributing/markup-format.md:249]: I'm not sure I fully get what this one does precisely

**Review by trolologuy — APPROVED** [review]

File: contributing/markup-format.md (hunk 1 above)
Comment (CollierCZ) [reply contributing/markup-format.md:249]: Compare:
* https://github.com/platformsh/platformsh-docs/blob/main/docs/src/guides/drupal9/deploy/deploy.md?plain=1#L23 => https://docs.platform.sh/guides/drupal9/deploy/deploy.html#data-migration
* https://github.com/platformsh/platformsh-docs/blob/main/docs/src/guides/micronaut/deploy/deploy.md => https://docs.platform.sh/guides/micronaut/deploy/deploy.html#data-migration

Comment (trolologuy) [reply contributing/markup-format.md:249]: Okey now I get it, with `{{ .Inner | .Page.RenderString }}` you tell **where** the optional additional text gets to be added in:
```
{{% reuse_markdown %}}
additional text
{{% /reuse_markdown %}}
```

**Comment by CollierCZ** [comment]
Handled in #2071

**Pull request closed without merging**